### PR TITLE
JLBP-6: Improving readability with positive wording

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Also, adding descriptions to the cases at the end, otherwise it is
not clear what the cases mean.

### CODE REVIEW

File: docs/JLBP-6.md [hunk 1]
```diff
@@ -156,18 +156,18 @@ is to the decision.
 Examples in open source
 -----------------------
 
-**Case 1**
+**Case 1 (Keep same Java package and Maven ID)**
```
Comment (elharo) [inline docs/JLBP-6.md]: can probably diowithout parentheses here and below

File: docs/JLBP-6.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ Consider the following renaming scenario:
 
 Given this scenario, here are the possible combinations of renamings:
 
-- **Don't rename Java package**:
-  - **Case 1: Don't rename Maven ID**. This approach can result in diamond
+- **Keep Java package**:
```
Comment (elharo) [inline docs/JLBP-6.md]: Keep java package the same? "Keep" alone doesn't seem to convey the full meaning

File: docs/JLBP-6.md (hunk 1 above)
Comment (garrettjonesgoogle) [reply docs/JLBP-6.md]: Removed the parentheses.

File: docs/JLBP-6.md (hunk 2 above)
Comment (garrettjonesgoogle) [reply docs/JLBP-6.md]: I changed it to "Keep Java package name", which I hope clarifies it.

**Comment by garrettjonesgoogle** [comment]
Addressed feedback, PTAL

File: docs/JLBP-6.md [hunk 3]
```diff
@@ -176,5 +176,5 @@ Examples in open source
 - JDOM (org.jdom -> org.jdom2)
 - jdeferred (org.jdeferred -> org.jdeferred2)
 
-**Case 5 (Bundle old and new in the existing Maven ID)**
+**Case 5 - Bundle old and new in the existing Maven ID**
```
Comment (elharo) [inline docs/JLBP-6.md]: Can we add a noun here? e.g Publish old and new packages with the existing Maven ID

Comment (garrettjonesgoogle) [reply docs/JLBP-6.md]: Good idea, updated.

**Pull request merged**